corehq/apps/analytics/tasks.py

```diff
@@ -887,7 +887,8 @@ def record_event(event_name, couch_user, event_properties=None):
     timestamp = unix_time_in_micros(datetime.utcnow())   # Dimagi KISSmetrics account uses UTC
 
     event_body = {
-        'client_id': couch_user._id,
+        'client_id': couch_user.userID,
+        'user_id': couch_user.userID,
```

Comment:
Is it worth leaving a comment about why it is necessary to add `user_id`? I doubt this is at risk of someone coming along and seeing what looks redundant and cleaning it up, but just a thought.

Reply:
07aab3fd0d45923a8772ca4e995f3a94c76c9021